upup/pkg/fi/cloudup/apply_inventory.go

```diff
@@ -0,0 +1,421 @@
+/*
```

Comment:
@andrewsykim I am guessing you missed this because I had \n errs :)  I fixed the style.  Let me know if you want to review, or wait for me to break this PR apart.